### ISSUE TITLE
I get error

### ISSUE THREAD

**Issue by firestorm23** [parmentf/node-ector#4]
node_modules/ector/node_modules/concept-network/lib/concept-network.js:66
    cb(null, this.node[id]);
    ^
TypeError: number is not a function
    at Object.ConceptNetwork.addNode = node_modules/ector/node_modules/concept-network/lib/concept-network.js:66:5)
    at Object.Ector.addEntry (node_modules/ector/lib/ector.js:134:30)
    at Object.<anonymous> (index.js:3:7)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:902:3

What is worng? I just installed clean package.

**Comment by firestorm23** [comment]
Looks like contept-network.js has been changed a lot, since last ECTOR modify.

**Comment by parmentf** [comment]
Oh, you're right. It's been a rather long time since I worked on `ector` and `concept-network`.

I thought I left `ector` in a correct state, and it seems not. :-1: 

It tries to use `concept-network` in a synchronous way, and it is written in an asynchronous one (it needs a callback as a parameter).

I think there is a previous version of  `concept-network` that should work.

Version 1.1 of `concept-network` should work [as expected](https://github.com/parmentf/node-concept-network/blob/v1.1.0/test/concept-network_test.js#L35).

Simply modify the expected version of `concept-network` in `package.json` from `^1.2.0` to `1.1.0`, I messed up with semantic versioning. My bad, sorry.

And thanks for saying.
